src/System.Data.Common/tests/System/Data/SqlTypes/SqlDecimalTest.cs

```diff
@@ -276,6 +276,52 @@ public void AdjustScale()
         }
 
         [Fact]
+        public void AdjustScaleForFlatZero()
+        {
+            SqlDecimal value = SqlDecimal.Parse("0");
+            Assert.True(value.Precision == 1);
```

Comment:
Since you aren't adding any error messages here, can you use Assert.Equal() for these equality tests?